mods/d2k/rules/defaults.yaml

```diff
@@ -268,7 +220,8 @@
 		VoiceSet: InfantryVoice
 
 ^Plane:
-	Inherits: ^GainsExperience
+	Inherits@1: ^ExistsInWorld
+	Inherits@2: ^GainsExperience
```

Comment:
None of the aircraft in D2k gain experience - only Carryalls are buildable, and those are non-combatant.